lib/delayed_job_web/application/views/job.erb

```diff
@@ -60,5 +60,11 @@
         <%=h job.failed_at %>
       </dd>
     <% end %>
+    <% job.attribute_names.excluding('id', 'priority', 'attempts', 'queue', 'handler', 'last_error', 'run_at', 'locked_at', 'locked_by', 'failed_at').each do |column| %>
+      <dt><%= column %></dt>
+      <dd>
+        <%=h eval("job.#{column}") %>
```

Comment:
Is there a security risk here someone could exploit with arbitrary code execution within the `eval()` block? Any safer alternatives to this with the same result?

I do it's scoped to `job.attribute_names`, but it feels like there's possible a safer alternative here.

Reply:
One idea (untested), is there externalized configuration (an initializer?) that can be added with the non-standard column names instead or using eval?

Reply:
This should be equivalent, I think.
```suggestion
        <%=h job.send(column) %>
```